### ISSUE TITLE
Continuing the plugin

### ISSUE THREAD

**Issue by ghost** [moomaxie/BetterShops#22]
@moomaxie 

Hi there, i'm interested in continuing your plugin.

You haven't been active in about 48 weeks and i've seen a lot of people who want support for this, and i'm happy to give it to them, but it turns out that I can't, due to the license you put on it.

I can get in contact via Skype, Discord, PM on Spigot, or simply here. I hope your great work can be continued!

Best Regards,
Athena

**Comment by LeafyMitsuwa** [comment]
From what I've seen, this looks like a plugin that might seem a nice addition to my server, however I'm unable to use it due to it not getting enabled.

Future support would be nice.

**Comment by ghost** [comment]
Yeah - i've seen this as a popular reason. A lot of people really like this plugin and say it's the best they've seen/have, but the lack of support is horrible, and I don't think I can legally modify & redistribute the code due to the license on the documents.

**Comment by dwi** [comment]
If you want to contribute to the plugin, you can start from this fork https://github.com/Majncraft/BetterShops

I already fixed countless duplication bugs and had to remove integrated NPC thingie, that wasn't just working right.

**Comment by ghost** [comment]
I've seen that fork a lot already, but I would rather make a version that can be legally distributed, noting the "All Rights Reserved" on documents

**Comment by Aztorius** [comment]
I'm really interested in an updated plugin with a fork.
@dewick the only thing your fork is missing is an **issue section**, can you add this ?
I've seen the "All Rights Reserved" and **we absolutely need an answer** from @moomaxie in order to change it to something like GPLv2.
I really enjoyed using this plugin.

**Comment by ghost** [comment]
Yeah, that's basically my logic there @Aztorius. All Rights Reserved should be treated like "All Rights Reserved"
